app/routes/__index/domains/index.tsx

```diff
@@ -75,10 +77,20 @@ export const action = async ({ request }: ActionArgs) => {
 };
 
 export default function DomainsIndexRoute() {
+  const revalidator = useRevalidator();
   const domains = useTypedLoaderData<typeof loader>();
+  const pending = domains.some((domain) => domain.status === 'pending');
```

Comment:
I don't *think* we need `useMemo` here, since we always want to be in sync with the value of `domains, and the cost of the calculation is cheap.

But I'll add and we can remove if it i's not working.